### PR TITLE
Add previous round bets

### PR DESCRIPTION
**Pull request by henryzbw**
Used to show last round result, totalbets[resultIndex] and userbets[resultIndex]

### CODE REVIEW

**Comment by dwalintukan** [comment]
The `roundBets` resolver is getting too convoluted with logic now. Let's just simplify this and return an array of all the amounts for each round.

```
type RoundBets {
  totalRoundBets: [String]
  userRoundBets: [String]
  singleTotalRoundBets: [[String]]
  singleUserRoundBets: [[String]]
}

singleTotalRoundBets: [
  [1,2,3], // index 0 = totals for round 0
  [4,5,6] // index 1 = totals for round 1
]

// same thing for singleUserRoundBets
```

**Comment by henryzbw** [comment]
I refine them to return roundbets for each round 
```
type RoundBets {
  singleTotalRoundBets: [[String!]]
  singleUserRoundBets: [[String!]]
}
```
I removed 
```
  totalRoundBets: [String!]
  userRoundBets: [String!]
```
since it only returns current round bets, so its duplicate now

**Comment by henryzbw** [comment]
Tried to fix the tests, but don't know why it fails. sometimes it can pass, but most of the time it fails

**Comment by dwalintukan** [comment]
@henryzbw hm seems to be bugged, just remove the roundBets from the schema-helper.

File: src/graphql/resolvers.js [hunk 1]
```diff
@@ -61,72 +61,48 @@ module.exports = {
...
-        // Fetch all bets for this round
-        const bets = await DBHelper.findBet({
+        // Fetch all bets for this event
+        let bets = await DBHelper.findBet({
           txStatus: TX_STATUS.SUCCESS,
           eventAddress: address,
-          eventRound: currentRound,
         });
 
+        const singleTotalRoundBets = []
```
Comment (dwalintukan) [inline src/graphql/resolvers.js]: run linter

File: src/graphql/resolvers.js [hunk 2]
```diff
@@ -61,72 +61,48 @@ module.exports = {
...
+          singleUserRoundBets.push(fill(Array(numOfResults), '0'));
+        }
+
+        const resultSet = await DBHelper.findOneResultSet({
+          txStatus: TX_STATUS.SUCCESS,
+          eventAddress: address,
+          eventRound: 0,
+        });
+
+        if (resultSet && resultSet.length != 0) {
```
Comment (dwalintukan) [inline src/graphql/resolvers.js]: !==

**Pull request merged**